src/web_app/html/index_template.html

```diff
@@ -28,6 +28,11 @@
   <link rel="canonical" href="{{ room_link }}">
 {% endif %}
   <link rel="stylesheet" href="/css/main.css">
+  <!-- callstats dependencies: http://www.callstats.io/api/#integrating-with-your-sdk -->
+  <script src="https://api.callstats.io/static/callstats.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/2.1.1/jquery.min.js"></script>
+  <script src="https://cdn.socket.io/socket.io-1.2.0.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/jsSHA/1.5.0/sha.js"></script>
```

Comment:
jssha and socket-io are now downloaded via NPM and included in the closure compiler step. callstats.io is the only external/dynamic resource.

We could download it in the build step using CURL and then include when compiling as well? Then ofc we need to push a new version of apprtc if we want new callstats.io changes.